ddp-workspace/projects/ddp-dsm-ui/src/app/tissue-page/tissue-page.component.ts

```diff
@@ -139,7 +139,7 @@ export class TissuePageComponent implements OnInit {
           value: v,
         }, null, 'participantId',
         this.participant.participant.ddpParticipantId, Statics.ONCDETAIL_ALIAS,  null,
-        localStorage.getItem(ComponentService.MENU_SELECTED_REALM), this.participant.data.profile['guid']
+        sessionStorage.getItem(ComponentService.MENU_SELECTED_REALM), this.participant.data.profile['guid']
```

Comment:
We absolutely can refactor this (component, service, etc), however it will take some additional time to do.  If we decide to go forward with this solution, I'll add a ticket for it.